### ISSUE TITLE
Favourites are empty

### ISSUE THREAD

**Issue by mattiasb** [Ippytraxx/gnome-twitch#128]
I'm following a couple of games and channels on Twitch but the Favourites view is empty. 
Setting log level to 256 (which I guess is all?) indicates that the following-list isn't loaded from Twitch. Is this a an unimplemented feature (but planned?) or are something buggy on my end?

**Comment by Ippytraxx** [comment]
Ye I just merged code to master that should fetch your follows from Twitch when you are logged in. I think it's gonna take a few iterations to get it right so this is just a stepping stone.

Before we go too deep, have you refreshed your login token? GT has to ask for new permissions to retrieve and modify your follows.

**Comment by mattiasb** [comment]
I'm running the 0.2 RPM on F23 currently. I'll test a build on master now.

**Comment by mattiasb** [comment]
Works fine on master, great work! :)

**Comment by jangernert** [comment]
Not for me. With "r504.a12dfea" gt crashes right at startup when I'm logged in to twitch. When I delete the oauth token and user name with dconf, everything works fine again. But only until I log in again, which makes the UI crash again instantly.

When starting with a configured twitch account:

```
[18:03:32] GNOME-Twitch - Message : "{GtApp} Loading chat settings"
[18:03:32] GNOME-Twitch - Message : "{GtApp} Startup"
[18:03:32] GNOME-Twitch - Message : "{GtApp} Activate"
[18:03:32] Gtk - Critical : "Unable to retrieve object 'info_bar_ok_button' from class template for type 'GtWin' while building a 'GtWin'"
[18:03:32] GLib-GObject - Critical : "g_object_bind_property_full: assertion 'G_IS_OBJECT (source)' failed"
[18:03:32] GLib-GObject - Warning : "invalid (NULL) pointer instance"
[18:03:32] GLib-GObject - Critical : "g_signal_connect_data: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed"
Segmentation fault (core dumped)
```

When starting without an account and logging in from the UI:
```
[18:04:27] GNOME-Twitch - Message : "{GtApp} Loading chat settings"
[18:04:27] GNOME-Twitch - Message : "{GtApp} Startup"
[18:04:27] GNOME-Twitch - Message : "{GtApp} Activate"
[18:04:27] Gtk - Critical : "Unable to retrieve object 'info_bar_ok_button' from class template for type 'GtWin' while building a 'GtWin'"
[18:04:27] GLib-GObject - Critical : "g_object_bind_property_full: assertion 'G_IS_OBJECT (source)' failed"
[18:04:27] GLib-GObject - Warning : "invalid (NULL) pointer instance"
[18:04:27] GLib-GObject - Critical : "g_signal_connect_data: assertion 'G_TYPE_CHECK_INSTANCE (instance)' failed"
Error loading module '/usr/lib/mozilla/plugins/libflashplayer.so': /usr/lib/mozilla/plugins/libflashplayer.so: wrong ELF class: ELFCLASS32
Error scanning plugin /usr/lib/mozilla/plugins/libflashplayer.so, /usr/libexec/webkit2gtk-4.0/WebKitPluginProcess returned 256 exit status
[18:06:01] GNOME-Twitch - Message : "{GtTwitchLoginDlg} Got username '$USER' from form"
[18:06:03] GNOME-Twitch - Message : "{GtTwitchLoginDlg} Successfully got OAuth token '$TOKEN'"
Segmentation fault (core dumped)
```

**Comment by mattiasb** [comment]
Looks like a flash issue to me. The world would be a better place without that POS.

**Comment by Ippytraxx** [comment]
@jangernert Can you please try deleting the build directory and re-initialising meson. It seems like that the UI files haven't been recompiled (it can't find info_bar_ok_button) which is wierd...

Alternatively if that doesn't work, could you please make a fresh clone of GT and compiling from there.

**Comment by jangernert** [comment]
@Ippytraxx thx, that did the trick. Btw: really nice touch to ask the user to migrate all the local favorites to the twitch account. Only thing: it moved them on the first click (gtkspinner appeared for a second), but the info-bar reappeared. So I clicked a second time make it go away for good.
@mattiasb no, that is something that pops up on every application that uses a webkitgtk+ webview on my system. I messed up my flash installation somehow^^ But you're right with the second part of your statement of course :D
